docs/compute-resources.md

```diff
@@ -84,7 +85,20 @@ spec:
       cpu: 2
 ```
 
-The following TaskRun will be rejected, because it configures both step-level and task-level compute resource requirements:
+Either requests or limits can be configured individually:
```

Comment:
What we specifically want to clarify here is that if no requests are set, the *task-level* requests will be set to the *task-level* limits

Reply:
right. misunderstood your original point. - quick update